paasta_tools/cli/utils.py

```diff
@@ -490,16 +490,21 @@ def run_paasta_serviceinit(subcommand, master, service, instancename, cluster, *
         delta = "--delta %s" % kwargs['delta']
     else:
         delta = ''
-    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s %s' % (
+    if 'stream' in kwargs and kwargs['stream']:
```

Comment:
execute_paasta_serviceinit_on_remote_master is used by a number of commands. Each of these commands has its set of arguments. Arguments that are not shared by these commands are made into kwargs. For example, app_id and delta are "paasta scale" only; verbose and stream are "paasta status" only. You may want to leave the verbose flag and the stream flag in kwargs.